src/platform.ts

```diff
@@ -216,34 +225,84 @@ export function getRuntimeId(platform: string, architecture: string, distributio
 export function getRuntimeDisplayName(runtime: Runtime): string {
     switch (runtime) {
         case Runtime.Windows_64:
-            return 'Windows';
         case Runtime.Windows_86:
+        case Runtime.Windows:
             return 'Windows';
         case Runtime.OSX:
         case Runtime.OSX_ARM64:
             return 'OSX';
-        case Runtime.CentOS_7:
-            return 'Linux';
-        case Runtime.Debian_8:
-            return 'Linux';
-        case Runtime.Fedora_23:
-            return 'Linux';
-        case Runtime.OpenSUSE_13_2:
-            return 'Linux';
-        case Runtime.SLES_12_2:
-            return 'Linux';
-        case Runtime.RHEL_7:
-            return 'Linux';
+        case Runtime.CentOS:
+        case Runtime.Debian:
+        case Runtime.ElementaryOS_0_3:
+        case Runtime.ElementaryOS_0_4:
+        case Runtime.ElementaryOS:
+        case Runtime.Fedora:
+        case Runtime.GalliumOS:
+        case Runtime.LinuxMint:
+        case Runtime.OpenSUSE:
+        case Runtime.OracleLinux:
+        case Runtime.SLES:
+        case Runtime.RHEL:
         case Runtime.Ubuntu_14:
-            return 'Linux';
         case Runtime.Ubuntu_16:
+        case Runtime.Ubuntu_18:
+        case Runtime.Ubuntu_20:
+        case Runtime.Ubuntu_22:
+        case Runtime.Ubuntu:
+        case Runtime.Linux:
             return 'Linux';
-        case Runtime.Linux_64:
-            return 'Linux';
-        case Runtime.Linux_86:
-            return 'Linux';
         default:
-            return 'Unknown';
+            throw new PlatformNotSupportedError(undefined, runtime);
+    }
+}
+
+/**
+ * Get the fallback runtimes.

```

Comment:
Bit more explanation here of what "fallback" means and how the values should be ordered (more specific -> least specific)

Reply:
done